subiquity/server/server.py

```diff
@@ -539,7 +541,7 @@ async def wait_for_cloudinit(self):
                 "cloud-init status: %r, assumed disabled",
                 status_txt)
 
-    def select_autoinstall_location(self):
+    def select_autoinstall(self):
```

Comment:
I'm not sure this method name is quite appropriate any more. But I don't have any better ideas.

Reply:
I sat here for a few minutes pondering a better name and I have nothing.  If we think of one later I'm open to changing it.